src/Components/AdvancedSearch/Internal/QueryBuilder/QueryBuilder.styles.ts

```diff
@@ -54,13 +55,19 @@ export const getStyles = (
         subComponentStyles: {
             searchButton: {
                 root: {
-                    display: 'block'
+                    display: 'block',
+                    width: 100
                 }
             },
             addButton: {
                 root: {
                     display: 'block',
-                    color: props.theme.palette.themePrimary
+                    color: props.theme.palette.themePrimary,
+                    padding: 0,
+                    marginLeft: 0
```

Comment:
why are we manipulating the spacing so much on a button? these are usually pretty standard.

Reply:
Had to remove styling to align with the primary button. Since designs called for aligning text to the edge of the primary button I had to remove the padding + margin applied.